src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -23,7 +23,7 @@ const ProfileCard = ({
   buttonOptions
 }: ProfileCardProps) => {
   const classes = useStyles();
-  const profileIcon = Jdenticon.toSvg(email, 28)
+  const profileIcon = Jdenticon.toSvg(email, 47)
```

Comment:
Jdenticon also has some options. Use backColor from Jdenticon insted of CSS so it doesn't render a white shape on a white background.

Reply:
And let's also just allow it to use the green color from the design.